js/apps/system/_admin/aardvark/APP/react/src/components/form/CreatableMultiSelectControl.tsx

```diff
@@ -48,3 +39,53 @@ export const CreatableMultiSelectControl = (props: InputControlProps) => {
     </FormikFormControl>
   );
 };
+
+const createValue = (value: string | string[]) => {
+  if (Array.isArray(value)) {
+    return value.map(value => {
+      return {
+        value,
+        label: value
+      };
+    });
+  }
+  return [
+    {
+      value,
+      label: value
+    }
+  ];
+};
+
+const getValue = ({
+  field,
+  options
+}: {
+  field: { value: string | string[] | undefined };
+  options?: readonly OptionType[];
+}) => {
+  if (!field.value) {
+    return [];
+  }
+
+  // if no options provided, it's a newly created value
+  if (!options || options.length === 0) {
```

Comment:
This will cause the same effect, what do you think?

```suggestion
  if (!options || !options.length) {
```